projects/packages/config/src/class-config.php

```diff
@@ -151,7 +151,7 @@ protected function ensure_feature( $feature ) {
 		/**
 		 * Fires when a specific Jetpack package feature is initalized using the Config package.
 		 *
-		 * @since 8.2.0
+		 * @since 1.1.0
```

Comment:
Are we going to add `@since-jetpack` here or only for hooks that predate the package creation?

Reply:
Yes, the idea was to add `since-jetpack` only when the `since` tag would not reflect the truth (the method existed before the package was created). What do you think?

Reply:
Makes sense to me. For things like developer.jetpack.com, we can have logic over on that side that parses out what plugin versions have what packages.